docs/index.rst

```diff
@@ -10,10 +10,10 @@ The fairlearn project seeks to enable anyone involved in the development of arti
 The code is `available from GitHub <https://github.com/fairlearn/fairlearn>`_.
 
 .. toctree::
+   :maxdepth: 1
 
-    fairlearn.exceptions
-    fairlearn.metrics
-    fairlearn.postprocessing
-    fairlearn.reductions
-    fairlearn.widget
-
+   auto_examples/plot_binary_classification_COMPAS
```

Comment:
Could we have an 'examples' entry at this level of the table of contents, and the notebook beneath it?

Reply:
Does this look better?

![ex](https://user-images.githubusercontent.com/1663864/78355266-b6823500-75ad-11ea-8363-9b481c11ad87.png)

Reply:
On the landing page, I'd suggest not opening up the TOC for each example. Just list the examples. Well, 'example' for now.

Reply:
ok, done.

Reply:
Could there just be an 'examples' link in the sidebar TOC too. Right now, the Notebook itself is listed there:
![image](https://user-images.githubusercontent.com/28632930/78362295-d999ee00-7587-11ea-9e00-1971ba943400.png)

Reply:
How does it look now?